astronomy/eclipse_test.cpp

```diff
@@ -250,28 +214,29 @@ class EclipseTest : public testing::Test {
         current_time - bisection_interval,
         current_time + bisection_interval);
     EXPECT_EQ(Sign(actual_contact_time - current_time),
-              Sign(time_error)) << NAMED(time_error);
+              Sign(time_error)) << NAMED(actual_contact_time - current_time);
```

Comment:
Isn't the expected, actual backwards?
`EXPECT_EQ` is very confusing because of that in the midst of `EXPECT_THAT`s, preference for `EXPECT_THAT(actual, Eq(expected))` (also, this sounds like what we'd want is a `SameSign` matcher).